internal/services/registry/storage_test.go

```diff
@@ -128,6 +135,12 @@ func (s *StorageTestSuite) Test_Mint_SyntheticDevice() {
 	}
 	s.MustInsert(&udi)
 
+	integration := &ddgrpc.Integration{Id: integrationID}
+	var intTokenID uint64
+	s.deviceDefSvc.EXPECT().GetIntegrationByTokenID(gomock.Any(), gomock.Any()).Return(integration, nil).Do(func(ct context.Context, arg uint64) {
```

Comment:
I think for this one you could just plug in `intTokenID` for the second argument, no?